recipes/spirv-headers/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SpirvheadersConan(ConanFile):
+    name = "spirv-headers"
+    homepage = "https://github.com/KhronosGroup/SPIRV-Headers"
+    description = "SPIRV-Headers"
+    topics = ("conan", "sprv", "vulkan")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+
+    license = "BSD"
```

Comment:
They actually have a modified BSD license, so I'm not sure how to name it

https://github.com/KhronosGroup/SPIRV-Headers/blob/0a7fc45259910f07f00c5a3fa10be5678bee1f83/LICENSE#L1

Reply:
"Modified-BSD" ?

Reply:
Actually, this isn't a BSD license at all. 

It is basically MIT, but they changed `the "Software"` to `the "Materials"` and the third paragraph is completely made up

We are trying to use SPDX license identifier which is not possible in this case 🤔  https://spdx.org/licenses/

Reply:
Can we just list it as MIT then?

Reply:
I'm not feeling comfortable calling something MIT when it isn't

Reply:
Cross-referencing: https://github.com/KhronosGroup/SPIRV-Headers/issues/144